test/unit/views/DisassociateUcrViewSpec.scala

```diff
@@ -16,40 +16,56 @@
 
 package views
 
+import base.Injector
 import forms.DisassociateKind.{Ducr, Mucr}
 import forms.DisassociateUcr
-import helpers.views.CommonMessages
+import models.cache.DisassociateUcrAnswers
 import org.jsoup.nodes.Document
 import play.api.data.Form
 import play.twirl.api.Html
-import views.spec.{UnitViewSpec, ViewMatchers}
+import views.html.disassociate_ucr
 import views.tags.ViewTest
 
 @ViewTest
-class DisassociateUcrViewSpec extends UnitViewSpec with CommonMessages with ViewMatchers {
+class DisassociateUcrViewSpec extends ViewSpec with Injector {
 
-  private val page = new views.html.disassociate_ucr(mainTemplate)
+  private implicit val request = journeyRequest(DisassociateUcrAnswers())
```

Comment:
Why request is implicit when we're using it explicitly?

Reply:
I was going to ask something similar - but it is used implicitly elsewhere I think

Reply:
messages require a request implicitly.